ql/src/java/org/apache/hadoop/hive/ql/optimizer/lineage/Generator.java

```diff
@@ -58,12 +67,63 @@
 public class Generator extends Transform {
 
   private static final Logger LOG = LoggerFactory.getLogger(Generator.class);
+  private static final String ALL = "ALL";
```

Comment:
wouldn't it better to create LiniageFilter enum?

Reply:
In that case the values in `HiveOperation` would be listed again.